src/main/java/io/buji/pac4j/subject/Pac4jPrincipal.java

```diff
@@ -70,4 +71,16 @@ public boolean equals(Object o) {
     public int hashCode() {
         return profiles != null ? profiles.hashCode() : 0;
     }
+
+	@Override
+	public String getName() {
```

Comment:
Yes, sorry about that. Eclipse editor was inserting tabs instead of spaces ... should be ok now